src/components/measureLanding/MeasureLanding.tsx

```diff
@@ -29,6 +29,8 @@ export default function MeasureLanding() {
   let navigate = useNavigate();
   const measureServiceApi = useRef(useMeasureServiceApi()).current;
   const [measureList, setMeasureList] = useState<Measure[]>([]);
+  const [myMeasures, setMyMeasures] = useState<number>(0);
+  const [allMeasures, setAllMeasures] = useState<number>(0);
```

Comment:
Not a big deal but the state variable name is little misleading, Do you think we could specify allMeasuresCount or something like that?

Reply:
Good idea! updated